templates/discord-message-bot.js

```diff
@@ -0,0 +1,28 @@
+var request = require('request');
+
+module.exports =
+    function (context, callback) {
+        var start = Date.now();
+        var hook = JSON.parse(context.body_raw) // gather some data from our webhook and parse it (you can enable auto parsing for webtasks too)
+
+        request({
+            headers: {
+                'Authorization': 'Bot -->discord token here<--', // put your Discord API bot token here
```

Comment:
It would be better to have all tokens and uris as secrets which are then access off of the secrets object. If you look at our stripe [example](https://github.com/auth0/webtask-editor-templates/blob/master/templates/stripe-payment.yaml#L89) you will see we have created secrets for `STRIPE_PUBLISHABLE_KEY` and `STRIPE_SECRET_KEY`. These are then accessed via secrets like [here](https://github.com/auth0/webtask-editor-templates/blob/master/templates/stripe-payment.yaml#L51).